tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -42,19 +48,40 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
-public class DeleteOffsetsConsumerGroupCommandIntegrationTest extends ConsumerGroupCommandTest {
+@Tag("integration")
+@ExtendWith(ClusterTestExtensions.class)
+public class DeleteOffsetsConsumerGroupCommandIntegrationTest {
+    private final ClusterInstance clusterInstance;
+    public static final String TOPIC = "foo";
+    public static final String GROUP = "test.group";
+
+    DeleteOffsetsConsumerGroupCommandIntegrationTest(ClusterInstance clusterInstance) {     // Constructor injections
+        this.clusterInstance = clusterInstance;
+    }
+
     String[] getArgs(String group, String topic) {
         return new String[] {
-            "--bootstrap-server", bootstrapServers(listenerName()),
+            "--bootstrap-server", clusterInstance.bootstrapServers(),
             "--delete-offsets",
             "--group", group,
             "--topic", topic
         };
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {"zk", "kraft"})
-    public void testDeleteOffsetsNonExistingGroup(String quorum) {
+    ConsumerGroupCommand.ConsumerGroupService getConsumerGroupService(String[] args) {
+        ConsumerGroupCommandOptions opts = ConsumerGroupCommandOptions.fromArgs(args);
+
+        return new ConsumerGroupCommand.ConsumerGroupService(
+                opts,
+                Collections.singletonMap(AdminClientConfig.RETRIES_CONFIG, Integer.toString(Integer.MAX_VALUE))
+        );
+    }
+
+    @ClusterTests({
+        @ClusterTest(clusterType = Type.ZK),
```

Comment:
Could we define `ClusterTestDefaults` at class-level?

Reply:
Yes, updated it. Thank you.